RELEASES.md

```diff
@@ -0,0 +1,42 @@
+# Releases
+
+The Tendermint specification follows [semantic versioning](https://semver.org/). Implementations of the Tendermint protocol are encouraged to either mirror versions or have some form of mapping for clear readibility of the state of the implementation. The `master` branch is used for active protocol development. Developers should be wary that the specifications are subject to change and should instead rely on the latest release. In the event of a security vulnerability, fixes may be backported to prior major releases but generally only the last major release is supported. The following documentation is provided to describe the release process.
```

Comment:
```suggestion
The Tendermint specification follows [semantic versioning](https://semver.org/). Implementations of the Tendermint protocol are encouraged to either mirror versions or have some form of mapping for clear readibility of the state of the implementation. The `master` branch is used for active protocol development. Developers should be aware that the specifications are subject to change and should refer to tagged releases rather than head. In the event of a security vulnerability, fixes may be backported to prior major releases, but generally only the last major release is supported. The following documentation is provided to describe the release process.
```